### PR TITLE
update installation doc

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Update installation instruction

## Changes

* Replace Centos by Rocky Linux
  * Replace yum by dnf
  * Install Test::More (provided by Test::Simple) and Module::Install using binary package
* Add missing Class::Accessor dependency ( #566)
* Merge debian and ubuntu sections

## How to test this PR

* Install the engine, run the tests

### CODE REVIEW

**Review by mattias-p — DISMISSED** [review]

**Review by mattias-p — APPROVED** [review]
LGTM

**Review by matsduf — DISMISSED** [review]

**Comment by blacksponge** [comment]
I merged develop to resolve the conflict, please re-review

File: docs/Translation-translators.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ translation.
...
 For the steps below you need to work on a computer with Git, Perl and Gettext.
-Select what OS you want to work on from the list below. Other OSs will also work, 
+Select what OS you want to work on from the list below. Other OSs will also work,
 but you will have to find instructions elsewhere.
 
-* CentOS
+* Rocky Linux
 
   Install the following:
   ```
```
Comment (matsduf) [inline docs/Translation-translators.md:26]: On line 27, should it `yum`, not `dnf`?

**Review by matsduf — CHANGES_REQUESTED** [review]
Please update the description with a reference to the PR that added the Class::Accessor dependency to the code to make it possible to trace back.

File: docs/Translation-translators.md (hunk 1 above)
Comment (mattias-p) [reply docs/Translation-translators.md:26]: I'll be going over the Rocky parts again anyway.

Comment (blacksponge) [reply docs/Translation-translators.md:26]: it should be `dnf`, good catch

**Review by mattias-p — DISMISSED** [review]

**Review by matsduf — APPROVED** [review]
I approve.

**Comment by matsduf** [comment]
@blacksponge, please resolve conflict so it can be merge.

**Review by mattias-p — APPROVED** [review]

**Review by matsduf — APPROVED** [review]

**Pull request merged**